pkg/resource/template/template_funcs.go

```diff
@@ -42,6 +45,20 @@ func addFuncs(out, in map[string]interface{}) {
 	}
 }
 
+// hashNodeName hashes the node name for ipv6 only systems.
+// This hash is then converted to ipv4 IP to be used as router id.
+func hashNodeName(nodeName string) string {
+	hash := sha256.New()
+	hash.Write([]byte(nodeName))
+	ipv6 := hash.Sum(nil)
```

Comment:
Nit: The result of `Sum()` is the value of the hash, suggest you rename `ipv6` to `hashBytes`

Reply:
ack

Reply:
Think you missed this change?